### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jlovejoy**
re-order some things into difference sections - thanks to @vmbrasseur  for the jumping off point!

Signed-off-by: Jilayne Lovejoy

### CODE REVIEW

**Review by swinslow — CHANGES_REQUESTED** [review]
A few minor typos, otherwise looks good to me once those are fixes!

File: DOCS/README.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
-The files in this folder help the community collaborate and provide a standard interface for both reviewing and recording decisions on whether to include a license in the list.
+The files in this folder contains documentation important to understanding the SPDX Licesne List, documentation explaining processes related, and templates to help the community collaborate and provide a standard interface for both reviewing and recording decisions on whether to include a license in the list.
```
[the remark this answers is not in the document]
Comment (jlovejoy) [reply DOCS/README.md]: fixed!

File: DOCS/README.md [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
 
-## Using the templates
+## Process explanations
 
-This folder includes two templates to help standardize the license review process.
+* [Request new license](request-new-license.md) - instructions for how to request a new license be reviewed for addition to the SPDX License List
+* [New license workflow](./new-license-workflow.md) - the workflow for adding approved new licenses to the SPDX License List
+* [Git usage](git-usage.md) - guidance for setting up and using Git on your own system to contribute to the SPDX License List on GitHub
+
+## License submission emplates
```
[the remark this answers is not in the document]
Comment (jlovejoy) [reply DOCS/README.md]: fixed

File: DOCS/README.md [hunk 3]
```diff
@@ -1,21 +1,26 @@
...
+
+This folder includes two templates to help standardize the license review and decision process.
+
+* [Submission reponse template](template-license-review-checklist.md) - a 'fill-in-the-blanks' template for responding to new license submissions
```
[the remark this answers is not in the document]
Comment (jlovejoy) [reply DOCS/README.md]: fixed

**Review by swinslow — APPROVED** [review]

**Pull request merged**